JS/javascript.js

```diff
@@ -0,0 +1,191 @@
+const myArray = [
```

Comment:
- [ ] Kindly ensure your array only contain data about the project and not the styling here is a sample of the expected array:
![image](https://user-images.githubusercontent.com/61408860/189692845-3c998451-6c36-4b8a-885d-08ae6d92247e.png)

Reply:
- [ ] As the previous code reviewer mentioned that you should follow the sample as in your array the objects don't have a link attribute.